### PR TITLE
Feature/connect to API

### PR DESCRIPTION
**Pull request by leolpaz**
### Bookstore - Connect to API

Added on this pull request:

- Connection to the API for add and remove function
- Connection to the API to update the state on initial load

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -11,8 +11,22 @@ const Book = (props) => {
...
 
+  const removeFromAPI = async (id) => {
+    await fetch(`https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/YM3PSnyKoYXp8BbKCQ5a/books/${id}`, {
+      method: 'DELETE',
+      headers: {
+        Accept: 'application/json',
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        item_id: `${id}`,
```
Comment (dasileker) [inline src/components/Book.js]: - [x] Also the same goes for the removal.

File: src/components/AddBook.js [hunk 2]
```diff
@@ -16,6 +16,21 @@ const Form = () => {
...
+    await fetch('https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/YM3PSnyKoYXp8BbKCQ5a/books', {
+      method: 'POST',
+      headers: {
+        Accept: 'application/json',
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify({
+        title: `${book.name}&&&${book.author}`,
+        category: `${book.category}`,
+        item_id: `${book.id}`,
```
Comment (dasileker) [inline src/components/AddBook.js]: - [x] The API should be in the redux to store the data and push it to the `action.types`, now you are using it like the JS module creating a **Json.stringify** to add it to the body.

ooo

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @leolpaz,** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/AddBook.js (hunk 2 above)
Comment (leolpaz) [reply src/components/AddBook.js]: Done, thanks for the review.

**Review by eliasecasta — APPROVED** [review]
## Status:  Approved 🟢

Hi, @leolpaz, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.



------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by leolpaz** [comment]
Thanks for the approval.

**Pull request merged**